README.md

```diff
@@ -1,23 +1,27 @@
 # Development Containers
 
-This repository is for issues to shape the direction of development containers and the dev container CLI.
+This repository is for issues to shape the direction of development containers and the development container CLI.
 
-## Contributing
+A development container allows you to use a Docker container as a full-featured development environment. It can be used to run an application, to separate tools, libraries, or runtimes needed for working with a codebase, and to aid in continuous integration and testing.
 
-The goal of this repository is to shape the properties and future of development containers. If you are interested in contributing, please check out the [How to Contribute](contributing.md) document.
+The Development Containers Specification seeks to find ways to enrich existing formats with common development specific settings, tools, and configuration while still providing a simplified, un-orchestrated single container option – so that they can be used as coding environments or for continuous integration and testing.
 
-## Providing Feedback
+The first format in the specification, devcontainer.json, was born out of necessity. It is a structured metadata format that tools can use to store any needed configuration required to develop inside of local or cloud-based containerized coding. While this metadata can be persisted in a devcontainer.json today, we envision that this same structured data can be embedded in images and other formats – all while retaining a common object model for consistent processing.
 
-If you run into an issue using the [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension, please use the [vscode-remote-release](https://github.com/microsoft/vscode-remote-release) repository. Issues related to dev container definitions can be reported in the [vscode-dev-containers repository](https://aka.ms/vscode-dev-containers).
+Beyond repeatable setup, these same development containers provide consistency to avoid environment specific problems across developers and centralized build and test automation services. The open-source CLI reference implementation can either be used directly or integrated into product experience to use the structured metadata to deliver these benefits. It currently supports integrating with Docker Compose and a simplified, un-orchestrated single container option – so that they can be used as coding environments or for continuous integration and testing.
```

Comment:
Maybe add a "Learn more" with a link to where the web content will be (via an aka.ms link)?

Reply:
Created and linked to https://aka.ms/dev-container-site (which points to https://microsoft.github.io/dev-container-spec/, where the web content will be once we're ready to migrate it). 

A broader question - what additional info do we suspect users will learn on the web content? I wonder if "You may also review the Development Containers Specification site" could be more accurate it we feel there's a lot of content overlap, so that users don't expect a ton of new info there (unless we think we'll add a ton more there).

Reply:
Yeah thinking about how LSP and other specs do it, they route people to the web content as a user friendly way to consume it for those not getting into the weeds of the spec.

If we were keeping this all as markdown, I'd have the README include links to the specific articles in Github instead... so sort of the same type of thing.  I just remember  I found out about the LSP web content initially from the README at microsoft/language-server-protocol that points there.

Reply:
That makes sense! I linked to the site with: "You may also review the [Development Containers Specification site](https://aka.ms/dev-container-site)" which I think will help the goal of raising awareness of the site, especially for those who'd prefer higher-level web content, without users thinking they _must_ review the site to learn more.